packages/dev/core/src/Cameras/Inputs/freeCameraMouseInput.ts

```diff
@@ -91,7 +91,8 @@ export class FreeCameraMouseInput implements ICameraInput<FreeCamera> {
 
                 const srcElement = <HTMLElement>evt.target;
 
-                if (p.type === PointerEventTypes.POINTERDOWN && (this._currentActiveButton === -1 || isTouch)) {
+                if (p.type === PointerEventTypes.POINTERDOWN && (this._currentActiveButton === -1 || isTouch) && this._activePointerId === -1) {

```

Comment:
Adding a comment to prevent merging until the test is added. resolve this conversation after :-)

Reply:
Test has been added